liquibase-core/src/main/java/liquibase/sqlgenerator/core/AddForeignKeyConstraintGenerator.java

```diff
@@ -80,6 +82,8 @@ public Sql[] generateSql(AddForeignKeyConstraintStatement statement, Database da
                 // see "REFERENCES Clause" in manual
             } else if ((database instanceof FirebirdDatabase) && "RESTRICT".equalsIgnoreCase(statement.getOnDelete())) {
                 //don't use
+            } else if (database instanceof SybaseDatabase) {
```

Comment:
Should SybaseASADatabase be here, too?

Reply:
I'm not sure. This PR was just for sybase support, not sybase asa so didn't look at that.